### PR TITLE
Add eventsize option to source

### PR DESCRIPTION
**Pull request by bjoshua**
Also updating travis tests.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -225,6 +229,7 @@
     source_type            => $source_type,
     source_bind            => $source_bind, 
     source_port            => $source_port, 
+    source_event_size      => $source_event_size,
```
Comment (teriyakichild) [inline manifests/init.pp]: This parameter doesn't exist in the "flume_repose::properties" class.

**Comment by teriyakichild** [comment]
The following lines need to be removed from the tests:

https://github.com/bjoshua/puppet-flume_repose/blob/11aa2ef4ee7dcbbbf51508285cd2f95ae011cc48/spec/classes/init_spec.rb#L17
https://github.com/bjoshua/puppet-flume_repose/blob/11aa2ef4ee7dcbbbf51508285cd2f95ae011cc48/spec/classes/init_spec.rb#L39

File: manifests/init.pp (hunk 1 above)
Comment (bjoshua) [inline manifests/init.pp]: Updated Amended commit submitted.

**Comment by bjoshua** [comment]
@teriyakichild Huzza, all tests passed, thanks for the assist!

**Pull request merged**